hack/conformance.sh

```diff
@@ -41,6 +41,11 @@ trap teardown EXIT
 
 run_cmd="./sonobuoy run --wait"
 
+SONOBUOY_FOCUS=${SONOBUOY_FOCUS:-}
+if [ -n "$SONOBUOY_FOCUS" ]; then
+    run_cmd+=" --e2e-focus ${SONOBUOY_FOCUS// /_}"
+fi
+
```

Comment:
Let's do a SONOBUOY_EXTRA_ARGS so we can put whatever we want, it has plenty or arguments.